charts/etcd/templates/etcd-statefulset.yaml

```diff
@@ -233,7 +233,7 @@ spec:
         - --snapstore-temp-directory={{ .Values.backup.snapstoreTempDir }}
         - --etcd-process-name=etcd
 {{- if .Values.etcd.heartbeatDuration }}
-        - --enable-member-lease-renewal=false
+        - --enable-member-lease-renewal=true
```

Comment:
Please also make `--enable-member-lease-renewal=true` for unit tests

Reply:
[here](https://github.com/gardener/etcd-druid/blob/master/controllers/etcd_controller_test.go#L1620) and [here](https://github.com/gardener/etcd-druid/blob/master/controllers/etcd_controller_test.go#L1205)